src/System.Device.Gpio/System/Device/Gpio/GpioController.Linux.cs

```diff
@@ -49,11 +49,11 @@ private static GpioDriver GetBestDriverForBoard()
                         //{
                         //    return new HummingBoardDriver();
                         //} 
-                        return new UnixDriver();
+                        return UnixDriver.InitUnixDriver();
```

Comment:
I'm wondering - would it be hard to move out some of those refactoring changes out of this PR? Those should be easy to review and would be much easier to merge (fine if you leave as is but might be easier to merge separately)